package.json

```diff
@@ -111,7 +111,9 @@
   },
   "overrides": {
     "@appuniversum/ember-appuniversum": {
-      "tracked-toolbox": "^2.0.0"
+      "tracked-toolbox": "^2.0.0",
+      "ember-concurrency": "^2.3.7",
```

Comment:
Is there a reason to override this rather than upgrading? I tested this PR with `^3.1.0` and it seems to work.